test/test-timeouts/test.js

```diff
@@ -14,3 +14,9 @@ test('timeout messages must be strings', async t => {
 	t.snapshot(error.message, 'error message');
 	t.snapshot(error.formattedDetails, 'formatted details');
 });
+
+test('timeouts are not blocked by console output', async t => {
+	const result = await t.throwsAsync(fixture(['console-output.js']));
+	const error = result.stats.getError(result.stats.failed[0]);
+	t.snapshot(error.message, 'timeout despite console output');
+});
```

Comment:
This test is in the wrong place. `test-timeouts` refers to `t.timeout()` but this is modifying the overall timeout behavior. A new `idle-timeout/` folder would be more appropriate.

(Can't recall whether there's existing tests under the old `test-tap` but I'm happy not to add there.)

Reply:
Went ahead and moved the test, thanks.